docs/admin/maintenance/noble_migration_prep.rst

```diff
@@ -116,6 +116,21 @@ Please run:
 and send the output to :ref:`Support <getting_support>` immediately, so we can diagnose
 the severity.
 
+Ethernet
+--------
+
+If this fails, it means that you have an ethernet interface that is not connected to a network.
+
+Double-check that your ethernet cable is connected to the correct port on your server. If you are not using recommended hardware, you may have multiple ethernet interfaces configured. In that event, we recommend disabling all but the one you are using for SecureDrop.
+
+In Ubuntu Server, this can be done by removing the additional interfaces in the ``00-installer-config.yaml`` configuration file. For example, you can run:
+
+.. code:: sh
+   
+     sudo nano /etc/netplan/00-installer-config.yaml
+     systemctl daemon-reload
```

Comment:
netplan isn't systemd-ified for better or worse, you need to run `sudo netplan apply` for it to take effect. Alternatively we can recommend `sudo netplan try`, which basically enables the config, but requires confirmation within 120s otherwise it rolls back (so it's harder to lock yourself out of a remote server by accident).

Also running `ip -brief addr show` to get a list of the interfaces and see which are marked as down.

Reply:
Thanks! Should now be fixed